specification/sections/05-events.md

```diff
@@ -147,22 +161,25 @@ See [provider initialization](./02-providers.md#24-initialization), [setting a p
 
 ### Event handlers and context reconciliation
 
-Providers built to conform to the static context paradigm feature an additional `PROVIDER_CONTEXT_CHANGED` event, which is used to signal that the global context has been changed, and flags should be re-evaluated.
+Providers built to conform to the static context paradigm feature two additional events: `PROVIDER_CONTEXT_CHANGE_PENDING` and `PROVIDER_CONTEXT_CHANGED`.
```

Comment:
```suggestion
Providers built to conform to the static context paradigm feature two additional events: `PROVIDER_RECONCILING` and `PROVIDER_CONTEXT_CHANGED`.
```
This is the naming used in the "types" table. I see a couple more instances of `PROVIDER_CONTEXT_CHANGE_PENDING` throughout the PR, which I think we need to update

Reply:
Nice catch. I will make sure these are all resolved.